### ISSUE TITLE
Recursos para pessoas que desejam participar ativamente das iniciativas do Training Center

### ISSUE THREAD

**Issue by woliveiras** [training-center/sobre#66]
## Problema

Percebo que muitas das iniciativas de educação fazem mais do que só exibir o modelo a seguir para criar algo, mas tem sempre uma sessão de treinamento para quem quiser se tornar educador ou educadora.

No nosso caso nós estimulamos a [mentoria](https://github.com/training-center/mentoria), [grupos de estudos](https://github.com/training-center/study-groups), etc. Porém não estamos treinando quem não sabe como fazer e deseja contribuir.

Estamos pecando em recursos de aprendizagem sobre esses assuntos. 

## Exemplo

A Microsoft possui um recurso para educadores(as) utilizarem tecnologia em sala de aula para capacitar seus alunos para o futuro. Algo parecido com o Code Club, porém para professores em tempo integral.

[Microsoft Education (pt-br)](https://www.microsoft.com/pt-br/education)

Nesta página mesmo eles possuem toda uma UX pensada em atrair as pessoas para sua iniciativa.

Chega, lá em baixo, na opção de se tornar Innovate Educator:

![selection_012](https://user-images.githubusercontent.com/4243601/35143446-ba249d3c-fce8-11e7-9f8b-43933db61cd9.png)

E logo abaixo tem o treinamento para educadores:

![selection_013](https://user-images.githubusercontent.com/4243601/35143478-d8958182-fce8-11e7-9c8b-394f74e26640.png)

Precisamos pensar em alternativas desse tipo para nossos projetos.

---

Já foi discutido algumas vezes que precisamos repensar o mentoria, por exemplo, para um formato onde treinamos a pessoa antes dela se tornar, de fato, mentora.

Seria algo desse tipo, porém o material está online.

Porém essa issue é generalista. Em todos os projetos precisamos disso. Devemos tentar, aqui, não focar no mentoria, mas em um modelo de ensino abstrato suficiente para conseguirmos usar em todos os projetos.

**Comment by jaqueventurim** [comment]
Cara, concordo totalmente contigo!
Até porque as pessoas acham que não sabe o suficiente para participar das iniciativas (tipo ser mentora).

Não sei quanto a um curso pra isso, mas acho que tem que ter algo para quebrar essa barreira do medo que a pessoa tem e também traçar um caminho do que precisa ser feito e os objetivos das inciativas.

**Comment by khaosdoctor** [comment]
O que é exatamente o que você está proponto @woliveiras? Que os projetos deveriam ensinar algo a mais ou treinar as pessoas?

**Comment by woliveiras** [comment]
Não são os projetos que devam ensinar, mas o contrário, mano @khaosdoctor 

Somos nós mesmos que não estamos fazendo um onboarding muito bom para as pessoas que querem nos ajudar, por exemplo no mentoria.

Seria legal ter um modelo de treinamento para quem deseja se tornar mentor(a) em nosso projeto.

Nos grupos de estudos, mesma coisa: é bom ter mais informações sobre o que é gerenciar um grupo de estudos, como fazer isso, dicas, o que pode dar errado, o que pode dar muito certo, como se portar em algumas situações, etc.

Tudo isso é papel nosso como mantenedores dos projetos, sabe?

Então, eu vejo como projetos que mais necessitam ou vão precisar de um modelo de treinamento, seja para participar ou para ajudar a gerenciar:

- mentoria
- study groups
- slack
- como contribuir nos repositórios
- dojos
- meetups
- help4papers

**Comment by khaosdoctor** [comment]
Eu acho boa a ideia, mas ai não teríamos que dispor de mais mão de obra do que temos atualmente?

**Comment by jaqueventurim** [comment]
hahahaha acho que sim @khaosdoctor 
taí uma grande questão e preocupante né :/

**Comment by woliveiras** [comment]
> Eu acho boa a ideia, mas ai não teríamos que dispor de mais mão de obra do que temos atualmente?

Infelizmente precisamos fazer o material. Se não fizermos as pessoas que entrarem depois vão fazer como as que fazem hoje: entrar, se perder e sair depois de um tempo.

Só precisamos tratar tudo como projetos, especificar as coisas bem certinhas e fazer de um jeito que as pessoas da comunidade consigam nos apoiar. A comunidade é a mão de obra no open source. :grimacing:  :grimacing:  :grimacing:

**Comment by khaosdoctor** [comment]
Na minha opinião devemos fazer o seguinte:

- Pegamos um time de, no máximo, 5 pessoas que devem ser muito ativas e devem ter uma liderança (eu sugiro que seja alguém do #core) e essa liderança tem que estar muito engajada com o projeto
- Esse time é o responsável solo por criar o material, um de cada vez
- A cada material finalizado vamos passando para os "coordenadores" de projeto, que são aquelas pessoas que estão encabeçando os projetos individuais
- Eles analisam e dão o ok ou sugestões de melhoria para os materiais
- Uma vez pronto, passa-se para o próximo e começa de novo.

O que acham?

**Comment by jaqueventurim** [comment]
por que não fazer ao contrário? o core analisa e as pessoas que coordenam cria? (quero entender seu raciocínio)

**Comment by WendellAdriel** [comment]
Concordo com a @jaqueventurim, o Core deveria ficar apenas com a parte de análise e feedback enquanto as outras pessoas (que podem ser os "coordenadores") elaboram os materiais.

Na verdade creio que deixar livre para todos os membros mais ativos ou membros que tenham mais experiência em determinado projeto colaborarem para criar esse material ainda ficaria melhor ainda!!! :wink:

**Comment by jaqueventurim** [comment]
siiim, porque qualquer pessoa que quiser/tiver tempo pode ajudar!
mas antes de liberar o material um grupo específico de pessoas analisam e publicam.

**Comment by khaosdoctor** [comment]
Então ai que eu acho que pode surgir um problema, porque as vezes notamos que não existe um engajamento muito grande de algumas pessoas, mesmo coordenadores, então se a liderança não está engajada o time não vai produzir nada...

O pessoal do Core é (teoricamente) o time mais engajado do CT, então acho que daria um up pra galera que está fazendo

**Comment by jaqueventurim** [comment]
entendo seu ponto de vista.

o meu receio aqui é parecer algo superior do tipo, só nós podemos fazer isso!

o que podemos fazer é algum membro do core puxar esse projeto e incentivar a galera, mas não por ser do core, entende? só por ser uma pessoa mais ativa mesmo.

**Comment by WendellAdriel** [comment]
Eu entendo seu ponto de vista @khaosdoctor, mas como a @jaqueventurim falou pode parecer que está excluindo uma parte do pessoal que participa. Então eu concordo que tenha que ser pessoas ativas e engajadas com o projeto para fazer essa parte, por isso mesmo teria de ter alguém mais ativo (provavelmente do Core) que fique com a responsabilidade de "coordenar" o andamento disso!!! xD

**Comment by khaosdoctor** [comment]
Então, mas ai eu fiquei confuso, você concorda comigo então?

**Comment by WendellAdriel** [comment]
Hahahahaahhhahah, ficou meio confuso mesmo @khaosdoctor...
Eu acho que pode abrir para outras pessoas ajudar, mas com supervisão de alguém (ou um grupo) de pessoas mais ativas, sendo elas do core ou não.

**Comment by khaosdoctor** [comment]
Ahhh ta, agora entendi, é possível sim, mas o meu medo é manter as pessoas incentivadas, que é o principal problema de comunidades em geral.

**Comment by WendellAdriel** [comment]
Sim sim, isso é um problema geral, mas devemos encontrar uma forma de manter o pessoal ativo. Eu mesmo me candidato a ajudar na criação desses materiais.
